content/docs/1.5.0/references/settings.md

```diff
@@ -225,7 +226,65 @@ Upgrade Checker will check for a new Longhorn version periodically. When there i
 
 The latest version of Longhorn available. Automatically updated by the Upgrade Checker.
 
-Only available if `Upgrade Checker` is enabled.
+> Only available if `Upgrade Checker` is enabled.
+
+#### Allow Collecting Longhorn Usage Metrics
+
+> Default: `true`
+
+Enabling this setting will allow Longhorn to provide valuable usage metrics to https://metrics.longhorn.io/.
+
+This information will help us gain insights how Longhorn is being used, which will ultimately contribute to future improvements.
+
+**Node Information collected from all cluster nodes includes:**
+- Number of disks of each device type (HDD, SSD, NVMe, unknown).
+  > This value may not be accurate for virtual machines.
+- Host kernel release.
+- Host operating system (OS) distribution.
+- Kubernetest node provider.
+
+**Cluster Information collected from one of the cluster nodes includes:**
+- Longhorn namespace UID.
+- Number of Longhorn nodes.
+- Number of volumes of each access mode (RWO, RWX, unknown).
+- Number of volumes of each data locality type (disabled, best_effort, strict_local, unknown).
+- Number of volumes of each frontend type (blockdev, iscsi).
+- Average volume size.
+- Average volume actual size.
+- Average number of snapshots per volume.
+- Average number of replicas per volume.
+- Average Longhorn component CPU usage (engine image, instance manager, manager, ui)
+- Average Longhorn component CPU usage (engine image, instance manager, manager, ui)
```

Comment:
I would suggest let's collect control-plane (longhorn-manager) and data-plane (instance manager) only. 

Why there are two CPU usage? I think one is the CPU request set, right?

Reply:
> I would suggest let's collect control-plane (longhorn-manager) and data-plane (instance manager) only.

Updated.

> Why there are two CPU usage? I think one is the CPU request set, right?

Meant memory. Corrected!